### DISCUSSION TITLE
How does the extension get external parameters

### DISCUSSION THREAD

**Discussion by pingbaohua** [eclipse-theia/theia#14070] [Q&A]
How to create a global static variable in theia and then get the variable in the extension application installed in the extension，and  cannot use settings.json to store this variable

**Comment by msujew** [comment]
Hey @pingbaohua,

Are you talking about VS Code extensions? From Theia's point of view they are black boxes. If they offer an API to set values like via commands or as a configuration item, you can call the appropriate Theia API.

Can you describe your use case a bit more?
